blog/2022/2022-04-03-new-home-for-the-loopback-blog.md

```diff
@@ -0,0 +1,115 @@
+---
+title: New Home for the LoopBack Blog
+date: 2022-04-03
+slug: new-home-for-the-loopback-blog
+authors: achrinza
+tags: [Meta]
+---
+
+After a long hiatus, the LoopBack Blog has gotten a new home! Learn how it came
+to be and how we managed to modernise blogging stack along the way.
+
+<!--truncate-->
+
+Since the early days of LoopBack, the blog was managed by a team in StrongLoop
```

Comment:
I'm thinking for the first paragraph, we talk a bit about why the blog site need a new home (which you did here) and what options we have. In the next paragraphs, we go a bit in details on each of the options and why we picked what we have today. 

Suggestion for this paragraph:

---
As part of the planning for LoopBack moving to OpenJS Foundation, there have been discussions around finding a new home for the blog, parting from the [strongloop.com](https://strongloop.com) web site. A few alternatives had brought up, such as having the blog posts as part of the loopback.io repo, using a third party blog site and creating a blog site in a separate repo.